wiki/blocks/first-block.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Adding your first Block
+index: 1
+---
+# Adding your first Block
+
+Creating a Block is pretty similar to creating an Item, [read the corresponding article](../items/first-item) for more elaborate explanations.
```

Comment:
```suggestion
Creating a Block is pretty similar to creating an Item, read [the corresponding article](../items/first-item) for more elaborate explanations.
```
not a fan of this entire sentence honestly (I don't like saying "x is just like y!" without giving any reasons), I'd prefer something more like:
```
Creating a Block is quite similar to creating an [item](../items/first-item), but now leveraging both the block *and* item registries, as well a more in-depth process for creating a model.
```